common/openapi.yml

```diff
@@ -145,9 +145,84 @@ paths:
           description: "Version not configured"
         '500':
           $ref: '#/components/responses/ServerError'
+  /api/secrets/sshkeys/v1/{provider}:
+    parameters:
+      - $ref: '#/components/parameters/sshkeyProviderParam'
```

Comment:
let's be consistent on case, is it sshKey or sshkey?

Reply:
i picked sshKey